### DISCUSSION TITLE
Error message

### DISCUSSION THREAD

**Discussion by christianaz** [FrandsenGroup/beams#101] [Q&A] [answered]
Hello! I keep getting this error message, but I'm not sure what it means. What is this error message?
![image](https://user-images.githubusercontent.com/84045977/137210941-7088bb2a-a18c-46af-8020-a462b61159f9.png)

**Comment by aPeter1** [comment]
Did this error message pop up when you tried to fit something?

**Reply by christianaz** [reply]
Yes. I was trying to do some global fits. The bin size was really small, which I think might be why it was causing problems (?)

**Reply by aPeter1** [reply]
Very possible, I'll open up an issue for this. How small?

**Reply by christianaz** [reply]
Well, I was trying to do one bin because I'm doing a comparison for a fit in musrfit where the bin size is one. I put it at 0.390625, but later tried 0.4 to no avail.

**Reply by aPeter1** [reply]
I was able to reproduce it (yay!), I don't need the files but were they all from the same or different experiments?

**Reply by christianaz** [reply]
Cool. They were all from the same experiment

**Comment by aPeter1** [comment, marked as answer]
So this should be fixed now with #102. I made an incorrect assumption in the code 👍

**Comment by christianaz** [comment]
Hello! I keep getting a new error message and I'm not sure what it means
![image](https://user-images.githubusercontent.com/84045977/152469712-81831b98-e717-481e-97bb-33df806e9469.png)

**Reply by aPeter1** [reply]
@christianaz could you send me the values in your config and batch tables. I'll see if I can figure it out

**Reply by christianaz** [reply]
<img width="184" alt="image" src="https://user-images.githubusercontent.com/84045977/152570729-4ab1dbea-8f3e-4b22-9b6c-58dea70d77d0.png">
<img width="184" alt="image" src="https://user-images.githubusercontent.com/84045977/152570817-58ff6f3c-a632-4150-a134-8a32289ac233.png">

alpha =1.08 and a= 0.23. I haven't done anything with the batch section. The first time it gave me the error I had put limits on f, but it's still giving me the error with the current values

**Reply by christianaz** [reply]
This message keeps showing up in the Anaconda prompt each time the error message pops up.

<img width="738" alt="image" src="https://user-images.githubusercontent.com/84045977/152571296-36990a95-6dda-41e2-a187-3c20bb09cfea.png">

**Comment by aPeter1** [comment]
@christianaz  Alrighty, this is now fixed with #154, we were raising negative values to fractional powers which is apparently illegal in all 50 states.
